### PR TITLE
Update to .NET 6

### PR DESCRIPTION
**Pull request by anthonychu**
Must merge at the same time as updates to Learn Module. https://github.com/MicrosoftDocs/learn-pr/pull/22574

### CODE REVIEW

**Review by codemillmatt — APPROVED** [review]

**Review by softchris — APPROVED** [review]

File: Api/local.settings.example.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+    "IsEncrypted": false,
+    "Values": {
+        "AzureWebJobsStorage": "",
+        "FUNCTIONS_WORKER_RUNTIME": "dotnet"
+    },
+    "Host": {
+        "LocalHttpPort": 7071,
+        "CORS": "*",
```
[the remark this answers is not in the document]
Comment (anthonychu) [reply Api/local.settings.example.json:9]: Currently the Learn module says use * locally but CORS config is not necessary in prod.

I added this file to make it easier if someone wants to use vs code instead. It’s not necessary for the Learn module. If you think this is confusing we can remove it.

**Comment by csharpfritz** [comment]
LGTM

**Review by csharpfritz — APPROVED** [review]

File: Api/local.settings.example.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (softchris) [reply Api/local.settings.example.json:9]: > 

I think more a note that it isn't used in prod, but used for local dev, could be helpful..

[the remark this answers is not in the document]
Comment (anthonychu) [reply Api/local.settings.example.json:9]: Any suggestions on where to include this note? This file shouldn't have comments. It's not used in the module but is useful if anyone wants to the repo outside of VS.

[the remark this answers is not in the document]
Comment (c0g1t8) [reply Api/local.settings.example.json:9]: Rename from _local.settings.example_ to _local.setting.forLocalDevelopment_. 😃

**Review by jamesmontemagno — APPROVED** [review]

**Pull request merged**